moveit_ros/planning/planning_pipeline/res/planning_pipeline_parameters.yaml

```diff
@@ -5,7 +5,12 @@ planning_pipeline_parameters:
     default_value: "UNKNOWN",
   }
   request_adapters: {
```

Comment:
Are these parameters something that can change at runtime with dynamic params? If so do we have to worry about someone changing the list while we are processing a request?

Reply:
No they are only loaded once during initialization and then they don't change